sky/serve/replica_managers.py

```diff
@@ -429,15 +429,17 @@ def probe(
                     f'code {response.status_code}')
             if response.status_code == 200:
                 msg += '.'
+                log_method = logger.info
             else:
                 msg += f' and response {response.text}.'
-            logger.info(msg)
+                log_method = logger.error
```

Comment:
For discussion: Shall we consider using colorama to make it yellow? Since controller emits tons of logs.